### ISSUE TITLE
Unable to generate AAR/framework - objcgen/Embeddinator4000.0.3.0

### ISSUE THREAD

**Issue by tuyen-vuduc** [mono/Embeddinator-4000#559]
I have tried both the nuget package and objcgen on my own project or a sample from @jamesmontemagno. 

I couldn't get  a iOS framework or a Android AAR file. I work on Mac with latest, stable VisualStudio by now.

```
Parsing assemblies...
    MonoAndroidToolsPath path is not found and no default location is provided; skipping
    Parsed 'HelloSharedUI.Droid.dll'
    Parsed 'FormsViewGroup.dll'
Processing assemblies...
Generating binding code...
    Generated: HelloSharedUI.Droid.h
    Generated: HelloSharedUI.Droid.c
    Generated: Mono.Android.h
    Generated: Mono.Android.c
    Generated: Java.Interop.h
    Generated: Java.Interop.c
    Generated: mscorlib.h
    Generated: mscorlib.c
    Generated: FormsViewGroup.h
    Generated: FormsViewGroup.c
    Generated: c-support.c
    Generated: c-support.h
    Generated: embeddinator.h
    Generated: glib.c
    Generated: glib.h
    Generated: mono-support.c
    Generated: mono-support.h
    Generated: mono_embeddinator.c
    Generated: mono_embeddinator.h
Compiling binding code...

Unhandled Exception:
System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.Max (System.Collections.Generic.IEnumerable`1[T] source) [0x00022] in <3fb92ec709f44a579da6a23087df403d>:0 
  at Embeddinator.XamarinAndroid+<>c.<.cctor>b__4_1 () [0x00029] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at System.Lazy`1[T].CreateValue () [0x00081] in <4fdc5ed61a074cafb49fa42deb20d521>:0 
  at System.Lazy`1[T].LazyInitValue () [0x00080] in <4fdc5ed61a074cafb49fa42deb20d521>:0 
  at System.Lazy`1[T].get_Value () [0x0003a] in <4fdc5ed61a074cafb49fa42deb20d521>:0 
  at Embeddinator.XamarinAndroid.get_ApiLevel () [0x00000] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.CompileNDK (System.Collections.Generic.IEnumerable`1[T] files) [0x00115] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.CompileNativeCode (System.Collections.Generic.IEnumerable`1[T] files) [0x000ba] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.CompileCode () [0x000a6] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.Run () [0x000a5] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.CLI.Main (System.String[] args) [0x00092] in <861966a3e3bd4f5cb470ae751f890033>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.Max (System.Collections.Generic.IEnumerable`1[T] source) [0x00022] in <3fb92ec709f44a579da6a23087df403d>:0 
  at Embeddinator.XamarinAndroid+<>c.<.cctor>b__4_1 () [0x00029] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at System.Lazy`1[T].CreateValue () [0x00081] in <4fdc5ed61a074cafb49fa42deb20d521>:0 
  at System.Lazy`1[T].LazyInitValue () [0x00080] in <4fdc5ed61a074cafb49fa42deb20d521>:0 
  at System.Lazy`1[T].get_Value () [0x0003a] in <4fdc5ed61a074cafb49fa42deb20d521>:0 
  at Embeddinator.XamarinAndroid.get_ApiLevel () [0x00000] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.CompileNDK (System.Collections.Generic.IEnumerable`1[T] files) [0x00115] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.CompileNativeCode (System.Collections.Generic.IEnumerable`1[T] files) [0x000ba] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.CompileCode () [0x000a6] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.Driver.Run () [0x000a5] in <861966a3e3bd4f5cb470ae751f890033>:0 
  at Embeddinator.CLI.Main (System.String[] args) [0x00092] in <861966a3e3bd4f5cb470ae751f890033>:0 
```

**Comment by jsonsugar** [comment]
我和你遇到一样的问题

**Comment by tritao** [comment]
Hey, this should be solved with latest release 0.4.0, but we've not been able to publish the NuGet yet. You can compile master for now.

**Comment by tuyen-vuduc** [comment]
@tritao Do you have a guide to compile the master locally on a Mac? I tried but not successful.
